reflex/app.py

```diff
@@ -493,16 +493,30 @@ def get_frontend_packages(self, imports: Dict[str, str]):
         Example:
             >>> get_frontend_packages({"react": "16.14.0", "react-dom": "16.14.0"})
         """
-        page_imports = [
+        page_imports = {
             i
             for i in imports
             if i not in compiler.DEFAULT_IMPORTS.keys()
             and i != "focus-visible/dist/focus-visible"
             and "next" not in i
             and not i.startswith("/")
             and i != ""
-        ]
-        page_imports.extend(get_config().frontend_packages)
+        }
+        frontend_packages = get_config().frontend_packages
+        _frontend_packages = []
+        for package in frontend_packages:
+            if package in get_config().tailwind.get("plugins", []):  # type: ignore
+                console.warn(
+                    f"Tailwind packages are inferred from 'plugins', remove `{package}` from `frontend_packages`"
+                )
+                continue
```

Comment:
I don't think we should skip it for now - we should still install it if they specify it in the `frontend_packages`.

Reply:
All it does is basically detect duplicates (packages declared in both `tailwing/plugins` and in `frontend_packages`)
so the package is installed at the end.